pkg/ci/ci.go

```diff
@@ -36,7 +36,7 @@ func (c *Common) SetImageName(imageName string) {
 
 func (c *Common) BuildName(name string) string {
 	if c.ImageName != "" {
-		fmt.Printf("Using %s as BuildName", name)
+		fmt.Printf("Using %s as BuildName\n", c.ImageName)
```

Comment:
Should we try to pass in the logger here instead of fmt.Printf'ing?

Reply:
Might be a good idea 👍